docs/cli.rst

```diff
@@ -1,11 +1,271 @@
-The ``cci`` command line
+The ``cci`` Command Line
 ========================
 
-Basic operation
+
+
+Basic Operation
 ---------------
 
-Getting help
-------------
+    .. note:: If you are new to working with command line interfaces, the `Install Visual Studio Code <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup/review-base-requirements-install-visual-studio-code?trail_id=build-applications-with-cumulusci>`_ module covers installing and opening a terminal window in Visual Studio Code.
+
+After installing CumulusCI, use the ``cci`` command in your terminal or command prompt to interact with it.
+
+On a macOS, access the terminal via Terminal.app. On Windows, open cmd.exe.
+
+To see all available commands, type ``cci`` in your terminal:
+
+.. code-block:: console
+
+    $ cci
+    Usage: cci [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    error    Get or share information about an error
+    flow     Commands for finding and running flows for a project
+    org      Commands for connecting and interacting with Salesforce orgs
+    project  Commands for interacting with project repository configurations
+    service  Commands for connecting services to the keychain
+    shell    Drop into a Python shell
+    task     Commands for finding and running tasks for a project
+    version  Print the current version of CumulusCI
+
+To get information on a specific command listed above, type ``cci <command>``. 
+
+    For example, if you would like more information on the ``task`` command, type ``cci task``:
+
+.. code-block:: console
+
+    $ cci task
+    Usage: cci task [OPTIONS] COMMAND [ARGS]...
+
+    Commands for finding and running tasks for a project
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    doc   Exports RST format documentation for all tasks
+    info  Displays information for a task
+    list  List available tasks for the current context
+    run   Runs a task
+
+There are even more subcommands available under ``cci task``.
```

Comment:
Consider moving to the next section ("Get --help") and framing as "The ``cci task`` command has many useful subcommands, such as ``cci task run --help``."

Reply:
@justinvholt - For clarity,  the term subcommand is in reference to commands that fall under another command. For example, running `cci task --help` or `cci task` both produce the same output, and show a list of available commands that fall under `cci task`. These commands (`doc`, `info`, `list`, `run`) are considered the sub-commands of the `cci task` command (which does nothing on its own. 

Maybe we should just use the word "command" instead of subcommand.

Options are provided to specific commands in the format of either a name/value pairs (i.e. --option-name value) or, in the case of a "flag" by just specifying the name of the flag (e.g. `--debug`).